TransactionMobile/TransactionMobile/App.xaml.cs

```diff
@@ -34,6 +35,16 @@ public partial class App : Application
         /// </summary>
         public static TokenResponse TokenResponse;
 
+        /// <summary>
+        /// The estate identifier
+        /// </summary>
+        public static Guid EstateId;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Change the visibility of 'EstateId' or make it 'const' or 'readonly'.](https://app.codacy.com/manual/stuart_ferguson1/MobileApplication/pullRequest?prid=5700460)